lnbits/core/views/api.py

```diff
@@ -148,6 +152,30 @@ async def api_update_wallet(
     return await update_wallet(wallet.wallet.id, name, currency)
 
 
+@api_router.delete("/api/v1/wallet")
+async def api_delete_wallet(
+    wallet: WalletTypeInfo = Depends(require_admin_key),
+) -> None:
+    await delete_wallet(
```

Comment:
Suggestion:
- if user has zero wallets after the `delete_wallet` operation then create a default empty wallet

Reply:
i do not want to create stuff on `GET` or `DELETE` request

Reply:
i would rather delete the account when the last wallet is deleted

Reply:
> i do not want to create stuff on `GET` or `DELETE` request

- I understand for `GET` but for `DELETE` write operations are OK